pynest/nest/lib/hl_api_info.py

```diff
@@ -274,11 +274,13 @@ def SetStatus(nodes, params, val=None):
     n0 = nodes[0]
```

Comment:
Small nitpick: could you move this into the if below?

Reply:
Of course!